libs/akita-ng-effects/src/__tests__/effect.decorator.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { Effect } from '@datorama/akita-ng-effects';
+import { of } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+describe('@Effect Decorator', () => {
+  it('should create ab effect with metadata', (done) => {
+    class TestClass {
+      @Effect()
+      testEffect$ = of('sample')
```

Comment:
Why we need `done` here? It's sync operation.

Reply:
As far as I'm concerned this is async.
The test will finish before the assertion inside the tap operator has run.
It results in a false positive once the fakeAsync is removed.
In this test the assertion inside the tap operator isn't crucial I guess.